_settings/mappings/history.json

```diff
@@ -77,25 +45,26 @@
       "type": "string"
     },
     "has_parent": {
-      "type": "string",
-      "index": "not_analyzed"
+      "type": "boolean"
```

Comment:
If this works as `boolean`, great.  I remember running into issues with it and ended up using `string`.